core/src/OAuth2/OAuth2Client.cs

```diff
@@ -98,7 +98,7 @@ public void AddBodyParameter(string key, string value)
             {
                 if (method == HttpMethod.Post)
                 {
-                    response = await HttpRequest.SendPost(endpointUri, _headers, _bodyParameters, requestContext);
+                    response = await HttpRequest.SendPost(endpointUri, _headers, new FormUrlEncodedContent(_bodyParameters), requestContext);
```

Comment:
>_bodyParameters [](start = 107, length = 15)

in other places you renamed this to body. should that be done here as well?

Reply:
OK this change is not strictly necessary now. I'm reverting back to its original form. #ConsiderItDone